.changeset/cold-clouds-hang.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/brevo-api": minor
+---
+
+Add `DeleteUnsubscribedContactsConsole` job to enable the deletion of blocklisted contacts. This job can be utilized as a cronjob to periodically clean up the blocklisted contacts.
+
+Add additional functions `findContacts` and `deleteContacts` in `brevo-api-contact-service-ts` to enable finding and deleting all contacts.
```

Comment:
In my opinion this information should be deleted, since these functions cannot be used outside the package since the brevoApiContactService is not exported so this information is not useful. @raphaelblum what is your opinion on this? Should this be included in the changeset?  

```suggestion
```

Reply:
yes, I'd say only the relevant info for the application that uses the packages should be in the changeset.